### PR TITLE
Update news fragments with makefile changes

### LINKED ISSUE #1602: 1063 news fragment needs updating with other simulator names
The newsfragment says that only questa has been affected, but new PRs have appeared to fix the other simulators too.

https://github.com/cocotb/cocotb/blob/master/documentation/source/newsfragments/1063.change.rst

We can do this once they all are merged, and should probably use `:issue:` markup inline for each one.

### PR DESCRIPTION
**Pull request by themperek**
GHDL and Riviera-PRO makefiles have been aligned with others.

Closes #1602

### CODE REVIEW

**Comment by eric-wieser** [comment]
I'd probably put all three in a single release note.

**Comment by themperek** [comment]
@eric-wieser Like this?

File: documentation/source/newsfragments/1063.change.rst [hunk 1]
```diff
@@ -1 +1 @@
-Questa now interprets relative paths to source files as relative to the Makefile, the same behavior used by other simulators.
+Questa, Aldec Riviera-PRO and GHDL now starts and saves results.xml file in Makefile location, the same behavior used by other simulators. (:pr:`1598`) (:pr:`1599`)
```
Comment (cmarqu) [inline documentation/source/newsfragments/1063.change.rst:1]:
```suggestion
Mentor Questa, Aldec Riviera-PRO and GHDL are now being started in the directory containing the Makefile and also save :file:`results.xml` there, the same behavior used by other simulators. (:pr:`1598`) (:pr:`1599`)
```

File: documentation/source/newsfragments/1063.change.rst [hunk 2]
```diff
@@ -1 +1 @@
-Questa now interprets relative paths to source files as relative to the Makefile, the same behavior used by other simulators.
+Mentor Questa, Aldec Riviera-PRO and GHDL are now being started in the directory containing the Makefile and also save :file:`results.xml` there, the same behavior used by other simulators. (:pr:`1598`) (:pr:`1599`)
```
Comment (eric-wieser) [inline documentation/source/newsfragments/1063.change.rst:1]:
```suggestion
Mentor Questa, Aldec Riviera-PRO and GHDL are now started in the directory containing the Makefile and also save :file:`results.xml` there, bringing them in line with the behavior used by other simulators. (:pr:`1598`) (:pr:`1599`)
```

**Pull request merged**